frontend/src/components/common/RadioImageSelectGroup.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import {
+  Box,
+  FormControl,
+  FormLabel,
+  HStack,
+  Image,
+  Text,
+  useRadio,
+  useRadioGroup,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+
+import customTheme from "../../theme";
+import { DonationSizeInterface } from "../pages/Scheduling/types";
+
+interface RadioImageSelectGroupProps {
+  value: string;
+  values: DonationSizeInterface[];
+  name: string;
+  isRequired?: boolean;
+  onChange: (arg0: any) => void;
+}
+
+const RadioImageSelectButton = (props: any) => {
+  const { getInputProps, getCheckboxProps } = useRadio(props);
+  const input = getInputProps();
+  const checkbox = getCheckboxProps();
+  const { children } = props;
+
+  const RadioButtonStyle = {
+    default: {
+      cursor: "pointer",
+      borderWidth: "1px",
+      borderRadius: "6",
+      borderColor: "hubbard.100",
+      boxShadow: "md",
+      padding: "12px",
+      color: "hubbard.100",
+      shadow: "none",
+    },
+    selected: {
+      bg: "cottonCandy.100",
+      borderColor: "raddish.100",
```

Comment:
I think the borderColor is actually something called Champagne (#8D2569), I've added it into my ticket so you don't need to change it but would be good to change to avoid the merge conflict later :)